staging/cse/windows/azurecnifunc.ps1

```diff
@@ -416,6 +417,22 @@ function New-ExternalHnsNetwork
         {
             $managementIP = $netIP.IPAddress
             $adapterName = $na.Name
+
+            Write-Log "Get node IPv4 address assigned to the adapter $($na.Name): $($managementIP)"
+            $nodeIPs += $managementIP
+
+            if ($IsDualStackEnabled) {

```

Comment:
kind of curious how we disercn this but ran out of time to go look it up.

Reply:
From https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/, it supports `comma-separated dual-stack IP addresse`. I think that we should add both ipv4 and ipv6 addresses since we support dualstack.

<html>
<body>
<!--StartFragment-->
--node-ip string
--
  | IP address (or comma-separated dual-stack IP addresses) of the node. If unset, kubelet will use the node's default IPv4 address, if any, or its default IPv6 address if it has no IPv4 addresses. You can pass "::" to make it prefer the default IPv6 address rather than the default IPv4 address.

<!--EndFragment-->
</body>
</html>